app/upgrade_test.go

```diff
@@ -56,6 +55,9 @@ func TestSkipOptimisticProcessingOnUpgrade(t *testing.T) {
 		Height: 1,
 	})
 
+	// Sleep for 1 second in case the ProcessProposalHandler goroutine is still running
+	time.Sleep(1 * time.Second)
+
```

Comment:
Slightly difficult to wait on the completion channel since that's accessing the optimistic processing info object too and I don't want to make any changes to the handler at this point